### ISSUE TITLE
Calls never complete

### ISSUE THREAD

**Issue by Frozenfire92** [Unity3dAzure/AppServicesDemo#1]
Hello,

I made a fork of bitrave [here](https://github.com/Frozenfire92/Unity3D-Azure-Mobile-Services-Example) and modified it but have experienced quite a few issues when it comes to building thus my search for something else. I found Unity3dAzure and it is good to see you are carrying on! I am more than interested to help development with this. Anyway onto the issue!

I tried cloning this repo and following the steps but none of the network requests ever seem to complete. Ignore the error, that is related to onGUI code from inputting a score

![capture](https://cloud.githubusercontent.com/assets/5893112/9297289/6a33114e-4473-11e5-80be-7ff6cb5ff262.PNG)

**Comment by deadlyfingers** [comment]
What platform are you running Unity on? I am adding support for Windows UnityEditor once I merge the 'windows' branch. This feature has been assigned as issue #2

**Comment by Frozenfire92** [comment]
I am running on Windows 10 (though can test on OSX if needed).

I was able to build to Android and have the calls successfully complete from the master branch.

From the windows branch I was able to build to .exe and android and have the calls complete in the editor. However I encountered compile errors when building to windows store / phone

**Comment by deadlyfingers** [comment]
Yes please try on Mac for now as Windows branch is not yet 100% ready. But let me know if you get it working on Mac so I can close this issue. I hope to have an update in soon!

**Comment by Frozenfire92** [comment]
Both the master and windows branch works on OSX in editor and mac .app build.

I got an on GUI error that didn't break anything (only on mac), I may bring over my new UI scene from [here](https://github.com/Frozenfire92/Unity3D-Azure-Mobile-Services-Example) if you are interested

**Comment by deadlyfingers** [comment]
Cool I will close off this issue. I've just pushed a commit into 'windows' branch that works in Windows RT app now so I'll issue a PR on that once I test changes on other platforms again. I was looking into the new Unity UI and that would be a good idea as the legacy GUI can get a bit flaky sometimes. I have created a new issue for this #3 if you want to fork it and do a PR. Although I will need to write up a Contributions doc to make sure this demo project works the way Unity developers expect / like.
